packages/block-library/src/post-terms/block.json

```diff
@@ -2,7 +2,7 @@
 	"$schema": "https://schemas.wp.org/trunk/block.json",
 	"apiVersion": 3,
 	"name": "core/post-terms",
-	"title": "Post Terms",
+	"title": "Terms",
```

Comment:
Just germs on its own also feels a little confusing to me. Which terms? Ones attached to the current post or terms in general?

Reply:
Is the post-terms block available, as-is? Or is it a wrapper for the actual terms blocks?

Reply:
Using the inserter or typing/terms` lets us choose between categories and tags.
But in the Site Editor Styles sidebar, we can select the Post Terms" block.

Reply:
I don't think Post Terms should be visible in the UI (it doesn't appear in inserters as noted). It's essentially a base for more contextual blocks referencing taxonomies etc. For reference if you insert it manually it doesn't do anything until you select a variation:

<img width="1263" alt="Screenshot 2023-08-10 at 11 37 10" src="https://github.com/WordPress/gutenberg/assets/846565/1d912cee-b6d8-4b69-95bc-f75979a8c810">

This makes it a more dev oriented block, and since developers are likely to intuit "Post" to mean _post type_ rather than _blog post_, perhaps it's okay to leave the title?

Reply:
Adding "Post Terms" back as the title.